### PR TITLE
Update composer.json

### PR DESCRIPTION
**Pull request by Glenderman**
Fixing error which stopped Stripe files being moved into src directory

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -12,7 +12,7 @@
 		],
 		"post-update-cmd": [
 			"echo 'Moving stripe into our src directory'",
-			"rm -rf ./src/site/libraries/stripe/*",
```
Comment (oshotton) [inline composer.json:15]: I would leave this line in. When updating `composer.json` and having to re-run the composer Docker container you will want to make sure the old stripe library is deleted before the new one is copied over.

Comment (Glenderman) [reply composer.json:15]: Ahh okay I see, I removed it since I thought it was deleting a directory which didn't exist, makes sense, should I put it before or after the new line of code?

Comment (oshotton) [reply composer.json:15]: The original error was:
```
error (error 1) - Script cp ./vendor/stripe/stripe-php/init.php ./src/site/libraries/stripe/init.php handling the post-update-cmd event returned with error code 1
```
This is complaining about the `cp ./vendor/stripe/stripe-php/init.php ./src/site/libraries/stripe/init.php` command on line 16. `./src/site/libraries/stripe` didn't exist so `init.php` couldn't be copied there. The `rm -rf ./src/site/libraries/stripe/*` command is deleting all files in the `./src/site/libraries/stripe` dir but not the dir itself (notice the asterisks (`*`)). Therefore it isn't the `rm` command that was the issue. Also worth pointing out is, `rm` doesn't fail if the thing you are trying to delete doesn't exist. You can test this for yourself:

```
$ ls /home/ubuntu/asdsadjikasd
ls: cannot access '/home/ubuntu/asdsadjikasd': No such file or directory
$ echo $?
2
$ rm -rfv /home/ubuntu/asdsadjikasd/*
$ echo $?
0
```

`echo $?` means print the exit code of the previous command. A zero exit code is OK, a non-zero exit code is FAIL.

TL;DR: It doesn't matter which command goes first. :)
